src/contracts/treasury/ICollector.sol

```diff
@@ -119,20 +119,16 @@ interface ICollector {
    **/
   function FUNDS_ADMIN_ROLE() external view returns (bytes32);
 
-  /**
-   * @notice Address of the current ACL Manager.
-   **/
-  function ACL_MANAGER() external view returns (address);
-
   /** @notice Returns the mock ETH reference address
    * @return address The address
    **/
   function ETH_MOCK_ADDRESS() external pure returns (address);
 
   /** @notice Initializes the contracts
    * @param nextStreamId StreamId to set, applied if greater than 0
+   * @param admin The default admin managing the FundsAdmins
    **/
-  function initialize(uint256 nextStreamId) external;
+  function initialize(uint256 nextStreamId, address admin) external;
```

Comment:
I doubt that it should be a part of the interface

Reply:
agreed, was there and i did not consider removing it - removed now